sql-plugin/src/main/spark330db/scala/com/nvidia/spark/rapids/shims/GpuCastShims.scala

```diff
@@ -16,6 +16,7 @@
 
  /*** spark-rapids-shim-json-lines
 {"spark": "330db"}
+{"spark": "332db"}
```

Comment:
This change looks out of place for this PR

Reply:
Yes, this change was merged to 23.08 in https://github.com/NVIDIA/spark-rapids/pull/8494 to fix the build.

Reply:
I've upmerged to pick up this change and remove from this PR